--- a/harmonia-cache/src/nar.rs
+++ b/harmonia-cache/src/nar.rs
@@ -168,6 +168,7 @@
                 let ranged_stream = create_range_stream(stream, offset, range_length);
 
                 return Ok(res
+                    .status(http::StatusCode::PARTIAL_CONTENT)
                     .insert_header((http::header::CONTENT_TYPE, "application/x-nix-archive"))
                     .insert_header((http::header::ACCEPT_RANGES, "bytes"))
                     .insert_header(cache_control_max_age_1y())
